client/extensions/wp-super-cache/controller.js

```diff
@@ -17,8 +17,8 @@ import WPSuperCache from './main';
 
 const controller = {
 
-	settings: function( context ) {
-		const siteId = getSiteFragment( context.path );
+	settings: function(context, next) {
+	    const siteId = getSiteFragment( context.path );
```

Comment:
@jsnmoon Is this something you've encountered, too? If so, do you happen to have found a fix?